st2common/st2common/runners/__init__.py

```diff
@@ -0,0 +1,207 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```

Comment:
Minor thing, I would prefer to put this into `base.py` file and leave __init__.py empty.